SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1419,10 +1423,12 @@ void USpatialReceiver::OnComponentUpdate(const Worker_ComponentUpdateOp& Op)
 
 	if (Category == ESchemaComponentType::SCHEMA_Data || Category == ESchemaComponentType::SCHEMA_OwnerOnly)
 	{
+		SCOPE_CYCLE_COUNTER(STAT_ReceiverApplyData);
```

Comment:
Should this be inside `ApplyComponentUpdate`?

Reply:
I want to differentiate between handover and data updates.

Reply:
Ah, I see 👍